### PR TITLE
FLOE-502: Integrate the responsive UIO.

### PR DESCRIPTION
**Pull request by cindyli**
The corresponding JIRA ticket: https://issues.fluidproject.org/browse/FLOE-502

### CODE REVIEW

File: accessibilitySprint2015.html [hunk 1]
```diff
@@ -11,49 +13,47 @@
...
-                    messagePrefix: "lib/infusion/framework/preferences/messages/",
-                    tocTemplate: "lib/infusion/components/tableOfContents/html/TableOfContents.html",
-                    template: "lib/infusion/framework/preferences/html/SeparatedPanelPrefsEditor.html"
-                });
+                floe.setupUIO();
             });
         </script>
 
-        <div id="skip" class="left">
-            <a href="#content">Skip to Content</a>
```
Comment (michelled) [inline accessibilitySprint2015.html]: Did you intend to remove this @cindyli?

File: index.html [hunk 2]
```diff
@@ -14,19 +15,12 @@
...
-        <link rel="stylesheet" type="text/css" href="lib/infusion/framework/preferences/css/fss/fss-theme-yb-prefsEditor.css" />
-        <link rel="stylesheet" type="text/css" href="lib/infusion/framework/preferences/css/fss/fss-theme-lgdg-prefsEditor.css" />
-        <link rel="stylesheet" type="text/css" href="lib/infusion/framework/preferences/css/fss/fss-text-prefsEditor.css" />
-        <link rel="stylesheet" type="text/css" href="lib/infusion/lib/jquery/ui/css/fl-theme-by/by.css" />
-        <link rel="stylesheet" type="text/css" href="lib/infusion/lib/jquery/ui/css/fl-theme-yb/yb.css" />
-        <link rel="stylesheet" type="text/css" href="lib/infusion/lib/jquery/ui/css/fl-theme-bw/bw.css" />
-        <link rel="stylesheet" type="text/css" href="lib/infusion/lib/jquery/ui/css/fl-theme-wb/wb.css" />
-        <link rel="stylesheet" type="text/css" href="lib/infusion/lib/jquery/ui/css/fl-theme-lgdg/lgdg.css" />
-        <link href="lib/infusion/framework/preferences/css/PrefsEditor.css" rel="stylesheet" type="text/css" />
-        <link href="lib/infusion/framework/preferences/css/SeparatedPanelPrefsEditor.css" rel="stylesheet" type="text/css" />
```
Comment (waharnum) [inline index.html]: It looks like this CSS was removed for debugging and then not restored?

File: index.html [hunk 3]
```diff
@@ -42,25 +36,34 @@
...
-                });
+                floe.setupUIO();
             });
         </script>
 
         <!-- BEGIN markup for Preference Editor -->
         <div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the mobile view.
+                A separate panel bar for mobile displays is needed to preserver the correct tab order.
```
Comment (waharnum) [inline index.html]: SP: "preserver" -> "preserve" (possibly copy-pasted from elsewhere, repeated in some other comments)

File: accessibilitySprint2015.html (hunk 1 above)
Comment (cindyli) [inline accessibilitySprint2015.html]: Good catch, @michelled. It has been added back.

**Comment by waharnum** [comment]
Hewlett logo area getting cut off at increased Text Size preferences:

image

The container has a fixed 3EM height, which should probably be adjusted to avoid the cut-off.

**Comment by waharnum** [comment]
One other comment (that I think would be a separate issue, since it is present on the site already): the Hewlett logo does not scale with text size adjustments or break well on a responsive layout.

I was speaking with Michelle and I think we can address this pretty quickly - I will make a separate JIRA / PR & self-assign.

File: index.html (hunk 2 above)
Comment (cindyli) [inline index.html]: The new set of css files for UIO is at line 18-23. Some of css files used for older version of UIO are no longer needed/existed in the preferences framework.

Comment (waharnum) [inline index.html]: Here is a side-by-side of the issues I am seeing with the images not adapting to the Colour & Contrast preference setting:

image

Comment (waharnum) [inline index.html]: It looks like some class names for swapping images based on UIO color/contrast themes may need to be updated in https://github.com/cindyli/floeproject.org/blob/FLOE-502/css/style.css#L510-L539

Comment (cindyli) [inline index.html]: Thanks for attaching pictures. The css files have been updated to use the new high contrast theme class names defined in the preferences framework. Images should adapt fine now.

Comment (waharnum) [inline index.html]: Works for me now. I'm good to merge this unless @michelled has further?

**Comment by cindyli** [comment]
Changed `height: 3em` used on the Hewlett logo area to `min-height: 3em` to make it flexible to adapt to larger texts.

All comments have been addressed. Ready for more reviews.

**Comment by waharnum** [comment]
@michelled, do you have anything further? Otherwise this looks good to merge to me.

**Comment by michelled** [comment]
@waharnum nope, nothing more - please merge.

**Pull request merged**